# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux/x86-64-isapnp.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-6.9/arch/x86/Kconfig.vanilla	2024-07-02 17:13:26.179497604 +0200
+++ linux-6.9/arch/x86/Kconfig	2024-07-02 17:14:03.286496125 +0200
@@ -2896,8 +2896,6 @@
 	  Enables ISA-style DMA support for devices requiring such controllers.
 	  If unsure, say Y.
 
-if X86_32
-
 config ISA
 	bool "ISA support"
 	help
@@ -2907,6 +2905,8 @@
 	  (MCA) or VESA.  ISA is an older system, now being displaced by PCI;
 	  newer boards don't support it.  If you have ISA, say Y, otherwise N.
 
+if X86_32
+
 config SCx200
 	tristate "NatSemi SCx200 support"
 	help
